components/ModEditSubmissionTopbar.vue

```diff
@@ -73,7 +75,37 @@ const acceptSubmission = () => {
     console.log('You are accepted')
 }
 
-const rejectSubmission = () => {
-    console.log('You are rejected')
+const rejectSubmission = async () => {
+    const facilityInputVariables = {
+        updateSubmissionId: selectedSubmissionId.value,
+        input: {
+            isRejected: true
+        }
+    }
+
+    const rejectSubmission = async () => {
+        try {
+            await graphQLClientRequestWithRetry(
+                gqlClient.request.bind(gqlClient),
+                rejectFacilitySubmissionGqlMutation,
+                facilityInputVariables,
+                60000,
```

Comment:
let's make an `options` object to make these variables more readable. We can make them be optional and have a default value.